Text/PrettyPrint/Leijen/Text.hs

```diff
@@ -516,6 +518,10 @@ string str = case T.uncons str of
                _                -> case (T.span (/='\n') str) of
                                      (xs,ys) -> text xs <> string ys
 
+stringStrict :: Ts.Text -> Doc
+stringStrict = mconcat . intersperse line . map textStrict . Ts.lines
+
```

Comment:
I can't believe I didn't do something like this for the normal `string` case... would you mind fixing that up as well please?